stacks-signer/src/runloop.rs

```diff
@@ -135,7 +138,7 @@ pub struct RunLoop<C> {
     pub blocks: HashMap<Sha512Trunc256Sum, BlockInfo>,
     /// Transactions that we expect to see in the next block
     // TODO: fill this in and do proper garbage collection
-    pub transactions: Vec<StacksTransaction>,
+    pub transactions: Vec<Txid>,
```

Comment:
Woah, I thought we were storing transactions from the stackerdb here?

Reply:
We do. have yet to get rid of this.